### ISSUE TITLE
Metric for max schematic size to successfully import/export?

### ISSUE THREAD

**Issue by CleanerOrc** [mcedit/mcedit2#37]
Is there any metric to determine max size of a region that can successfully be exported and then imported to another world using MCEDIT 1.3.0.0 or later?

Is import/export size related to available RAM size?   

Running on Win 7 with 8GB RAM.

**Comment by codewarrior0** [comment]
Read the contributing guidelines before posting an issue.

There is no practical limit to the size of exports in MCEdit 1.0. When the export size exceeds a certain limit (set as the "Block Buffer" setting), MCEdit switches from using .schematic files to a different export format based on the Minecraft world format, which has unlimited size. The only thing that limits you is how long you are willing to wait for a large copy to complete.

**Comment by CleanerOrc** [comment]
Dave, thanks for the info.  I was exporting a 300 Meg Block selection.  When I tried to import that to another world, I kept getting some kind of temp file not found error every time.  But I got it working by breaking into 3 different sections.

My question though, if it saved it in a different format, do I need to do anything different when I import it, or does MCEdit recognize that it’s a diff format file internally.

Thanks
